packages/react-components/react-slider/library/docs/Spec.md

```diff
@@ -2,98 +2,311 @@
 
 ## Background
 
-The Slider component allows users to quickly select a value by dragging an icon across a bar. It is often used when setting values with a relaxed precision such as audio volume and screen brightness.
+The `@fluentui/react-slider` package provides slider-style inputs for picking approximate values on a rail. It contains:
 
-## Variants
+- **Slider** – a single-thumb control for choosing one value (volume, brightness, etc.).
+- **RangeSlider** – a dual-thumb control for selecting a bounded range (price filters, min/max constraints, etc.).
 
-- `vertical` displayed vertically.
-- `disabled` read-only and does not update state.
+Both components follow the same visual language and share similar interaction patterns, but RangeSlider introduces a second thumb, collision detection, and two values exposed as `{ start, end }`.
 
-## API
+## Components
 
-### Slots
+- `Slider`: single-value input, identical to the existing v9 implementation.
+- `RangeSlider`: two-thumb range selection input that outputs `{ start, end }` values.
 
-- `root` - The outer container that wraps the slider structure.
-- `rail` - The Slider's base. It is used to visibly display the min and max selectable values.
-- `thumb` - The draggable icon used to select a given value from the Slider. This is the element containing `role = 'slider'`.
-- `input` - The hidden input for the Slider. This is the PRIMARY slot: all native properties specified directly on `<Slider>` will be applied to this slot, except `className` and `style`, which remain on the root slot.
+## Sample Code
 
-### Props
+### Slider
 
-See API at [Slider.types.ts](./src/components/Slider/Slider.types.ts).
+```tsx
+import { Slider } from '@fluentui/react-slider';
+import { Label, useId } from '@fluentui/react-components';
 
-## Structure
+const SliderExample = () => {
+  const sliderId = useId();
+  return (
+    <>
+      <Label htmlFor={sliderId}>Brightness</Label>
+      <Slider defaultValue={50} min={10} max={100} id={sliderId} />
+    </>
+  );
+};
+```
+
+### RangeSlider
+
+```tsx
+import { RangeSlider } from '@fluentui/react-slider';
+import { Label, Field, useId } from '@fluentui/react-components';
+
+const BasicRangeSlider = () => {
+  const labelId = useId();
+  return (
+    <>
+      <Label id={labelId}>Price range</Label>
+      <RangeSlider aria-labelledby={labelId} defaultValue={{ start: 20, end: 80 }} />
+    </>
+  );
+};
+
+const FieldRangeSlider = () => (
+  <Field label="Temperature">
+    <RangeSlider defaultValue={{ start: 20, end: 80 }} vertical />
+  </Field>
+);
+
+const ControlledRangeSlider = () => {
+  const [value, setValue] = React.useState({ start: 25, end: 75 });
+  return <RangeSlider value={value} onChange={(ev, data) => setValue(data.value)} min={0} max={100} step={5} />;
+};
+```
+
+---
+
+## Slider Component
+
+### Slider Variants
```

Comment:
I think you do not need to explicitly have Slider / RangeSlider for each heading. Second level headings for Slider and RangeSlider sections should be enough

Reply:
Right, then let's keep as is